File: Rimon/LeetCode/448_Find_all_numbers_dissappeared.cs
public class Solution {
    public IList<int> FindDisappearedNumbers(int[] nums) {
        var l= new List<int>();
        for(int i = 0; i<nums.Length; i++){
            var n = Math.Abs(nums[i]);
            if(nums[n-1] > 0)
                nums[n-1] *= -1;            
        }
        for(int i =0; i<nums.Length; i++){
            if(nums[i] > 0){
                l.Add(i+1);
            }
        }
        return l;
    }
}

File: Rimon/LeetCode/448_Find_all_numbers_dissappeared.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        
        i = 0
        while i<len(nums):
            n = nums[i]
            if nums[i] != nums[n-1]:
                nums[i],nums[n-1] = nums[n -1],nums[i]
            else:
                i+=1 
        l = []
        for i in range(len(nums)):
            if nums[i] != i+1:
                l.append(i+1)
        return l
                